src/views/v2/commerce/checkout/header.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Tickets Commerce: Checkout Page Header
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/commerce/checkout/header.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    https://evnt.is/1amp Help article for RSVP & Ticket template files.
+ *
+ * @since   TBD
+ *
+ * @version TBD
+ *
+ * @var \Tribe__Template $this                  [Global] Template object.
+ * @var Module           $provider              [Global] The tickets provider instance.
+ * @var string           $provider_id           [Global] The tickets provider class name.
+ * @var array[]          $items                 [Global] List of Items on the cart to be checked out.
+ * @var string           $paypal_attribution_id [Global] What is our PayPal Attribution ID.
+ * @var array[]          $sections              [Global] Which events we have tickets for.
+ */
+
+// @todo @bordoni @juanfra: Maybe move the modify attendees link to ET+.
+?>
+<div class="tribe-tickets__commerce-checkout-header">
+	<h3 class="tribe-common-h2 tribe-tickets__commerce-checkout-header-title">
+		<?php esc_html_e( 'Purchase Tickets', 'event-tickets' ); ?>
+	</h3>
+
+	<div class="tribe-common-b2 tribe-tickets__commerce-checkout-header-links">
+		<a
+			class="tribe-common-anchor-alt tribe-tickets__commerce-checkout-header-link-modify-attendees"
+			href="#"
+		><?php esc_html_e( 'modify attendees', 'event-tickets' ); ?></a>
```

Comment:
is the `href` going to have an actual URL or is this to act like a button?

Reply:
Yes, this one will actually come from ET+ as the idea is that it should take you to the AR page to modify the attendee information. Leaving this one here because G asked to mock-up things that he (or the team) will move later to ET+.